src/components/LayerCollection/LayerPO.js

```diff
@@ -90,13 +90,12 @@ function LayerPO({
   const openOrderErrorModalShow = useCallback(setUpdateOrderError, []);
 
   const updatePO = useCallback(values => {
+    const poNumber = getFullOrderNumber(values) || undefined;
+
     setIsLoading(true);
-    setSavingValues(values);
-    const saveFn = values.id
-      ? updateOrderResource
-      : createOrderResource;
+    setSavingValues({ ...values, poNumber });
```

Comment:
but that means in case of error `poNumber` will be filled with prefix and suffix. no?

Reply:
IMHO, you should keep `createOrderResource` and maybe it makes sense to call it `createOrEditOrderResource`, also change name of its first param from `order` to `orderFormValues` might help for readability

Reply:
fixed